src/iUtilities/iutilities.cpp

```diff
@@ -78,15 +78,19 @@ extern "C" {
 	{
 		return dataHandle<double>(i);
 	}
+	void* size_tDataHandle(size_t i)
+	{
+		return dataHandle<size_t>(i);
+	}
```

Comment:
is this actually used anywhere @evgueni-ovtchinnikov?

Reply:
`size_tDataHandle` has not been needed yet (BTW, if it ever does, the prototype must be added to `iutilities.h`). 

other things like that (`floatDataHandle` etc.) are used in `*_params.py` scripts

BTW have you taken notice of my discovery that a `pet.ImageData` object can be made contiguous in Python by e.g. adding 0? Can be used as a quick fix while a proper C++ fix is being sought for `VoxelsOnCartesianGrid(exam_info_sptr_v, proj_data_info,...)::grow(range)`.

Reply:
Why do you need `floatDataHandle` when you can use `dataHandle<float>`?

Reply:
because I use it in Python:
```
def set_float_par(handle, group, par, value):
    h = pyiutil.floatDataHandle(float(value))
    set_parameter(handle, group, par, h, inspect.stack()[1])
    pyiutil.deleteDataHandle(h)
```

Reply:
Now that we've dropped MATLAB, you could do this in the SWIG interface file instead, see https://swig.org/Doc4.1/SWIGPlus.html#SWIGPlus_template_directive
```
%include "sirf/iUtilities/DataHandle.h"
%template floatDataHandle dataHandle<float>;
``` 
I'd prefer that, but certainly not urgent. Maybe create a new issue for it and assign it somewhere for a late milestone!